extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -81,9 +80,6 @@ AmpBaseCarousel['props'] = {
   'loop': {attr: 'loop', type: 'boolean'},
 };
 
-/** @override */
-AmpBaseCarousel['shadowCss'] = scrollerStyles;
```

Comment:
We can just put the string CSS value here. e.g. `AmpBaseCarousel['shadowCss'] = '....'`, with a TODO to fix. Then this pull request could be mergeable sooner.

Reply:
I could take this idea even a step further, and put it the entire compiled css file :).

Reply:
Yep. Again, only if you want to merge before the build step is done. Just given you options given my OOO.